resume.md

```diff
@@ -1,49 +1,58 @@
-# FIRST LAST NAME
+# Tejesh Bhaumik
 
-(123) 456 - 7890 | email@gmail.com
+(123) 456 - 7890 | tejeshbhaumik@gmail.com
 
-www.website.com | github.com/myusername | linkedin.com/myusername
+| github.com/Tejesh2001 | linkedin.com/TejeshBhaumik
 
 ## EDUCATION
-**University of College** | B.S. Degree ... *MONTH 20XX*
-* **Coursework**: Data Structures, Algorithms, Upper Division CS Elective, Math Course, Statistics, Data Science, ETC.
+**University of College** | B.S. of Computer Science + Advertising... *May 2023*
+* **Coursework**: Data Structures, Algorithms, Advanced Linear Algebra, Statistics, Data Science, Probability, Brand Strategy.
 
 ## SKILLS
-* **Languages**: Python, Java, C, SQL, ...
-* **Libraries/Frameworks**: Pandas, TensorFlow, PyTorch, ...
-* **Version Control**: Git, Bitbucket, Sourcetree, GithubDesktop
+* **Languages**: Python, Java, C++, SQL, NodeJs, Android
+* **Libraries/Frameworks**: Pandas, Numpy, Scipy, Sympy
+* **Version Control**: Git, GithubDesktop
 * *...*
 
 ## EXPERIENCE
-**Software Engineer**: Company | City, State ... *Month 20XX - Present*
-* Implemented XYZ Microservice using TECHNOLOGY to delivery PRODUCT to 123 Customers Globally
-* Architected CORE INFRASTRUCTURE LIBRARY used across X teams to BLAH BLAH
-* I did X for Y to accomplish Z
-* ...
+**Android Developer**: Crowd Dynamics Lab   | Remote *September 2020 - Present*
+* Assist in the implementation of an Android tracer app using IoT principles
```

Comment:
What is the purpose of this app?   Why is this important and what problems does this app solve?

Be specific - what screens did create?  What tools did you use (Android Studio, any interesting APIs, recycler views? Drop your Android Specific background here).

How did you use IoT principles?